TTHAnalysis/python/plotter/susy-sos/scripts/htcondor_submitter.sh

```diff
@@ -0,0 +1,8 @@
+executable      = [SCRIPT]
+arguments       = [NAME]
+output          = [DIR]/job.[NAME].0.out
+error           = [DIR]/job.[NAME].0.err
+log             = [DIR]/job.[NAME].log
++JobFlavour	= "[QUEUE]"
++AccountingGroup = "group_u_CMST3.all"
```

Comment:
I am not sure what this option does. Maybe it is related with some permissions that some people may not have?